client/src/screens/events/Detail.js

```diff
@@ -274,12 +273,13 @@ class Detail extends Component {
         ))}
             {mapEventLocations.map(marker => (
               <Marker
```

Comment:
Missing "key" attribute - you'll get "Warning: Each child in an array..."

Reply:
doh I noticed that too but assumed it wasn't related :/